frontend/src/components/Form.js

```diff
@@ -41,11 +41,41 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Validates from inputs on submission and alerts user on error
+ * @return {Boolean} true or false based on validity of inputs
+ * @param {String} role
+ * @param {String} productArea
+ * @param {Number} duration
+ * @param {Number} timeAvailableUntilMilliseconds
+ * @param {Number} currentTimeMilliseconds
+ */
+export function validateFormInputs(role, productArea, duration,
+    timeAvailableUntilMilliseconds, currentTimeMilliseconds) {
+  const durationMilliseconds = duration * 60000;
+
+  if (isNaN(timeAvailableUntilMilliseconds)) {
+    alert('Please select a valid date.');
+    return false;
+  } else if (currentTimeMilliseconds + durationMilliseconds >=
+    timeAvailableUntilMilliseconds) {
+    // Check if a meeting is possible with the provided inputs
+    alert('Please select an larger time availability window');
+    return false;
+  } else if (productArea === '' || role === '') {
+    alert('Please select options for all form fields to submit!');
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
tiny nit:

no need to include an else here since all the if-branches return i.e. this can just be `return true` outside of an else {}

Reply:
Good point, thanks for the suggestion!